src-tauri/src/output_loop.rs

```diff
@@ -255,11 +277,24 @@ impl OutputLoopManager {
                     // Render DMX
```

Comment:
There is a lot of duplicated logic between this loop and the sACN loop below. Given they are both DMX renders, can we share the logic between the loops?